setup.py

```diff
@@ -20,7 +20,7 @@
 UTK_CMAKE_ARGS = [
     f"-DUTK_LOG=ON",
     f"-DUTK_USE_OPENMP=ON",
-    f"-DUTK_USE_FFTW=ON", 
+    f"-DUTK_USE_FFTW=OFF", 
```

Comment:
disabling FFTW from python binding?

Reply:
Indeed, that is a mistake... On my computer, the conda environement I use do not like fftw for a reason I still don't know...